applications/adv_networking_bench/cpp/CMakeLists.txt.install

```diff
@@ -0,0 +1,92 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.20)
+project(adv_networking_bench CXX CUDA)
+
+# Dependencies
+find_package(holoscan 2.6 REQUIRED CONFIG PATHS "/opt/nvidia/holoscan")
+find_package(holoscan-networking REQUIRED CONFIG PATHS "/opt/nvidia/holoscan")
+include(FetchContent)
+FetchContent_Declare(
+  MatX
+  GIT_REPOSITORY https://github.com/NVIDIA/MatX.git
+  GIT_TAG v0.8.0
+)
+FetchContent_MakeAvailable(MatX)
+
+# Global variables
+set(CMAKE_CUDA_ARCHITECTURES "70;80;90")
+
+# Create the executable
+add_executable(adv_networking_bench
+  kernels.cu
+  main.cpp
+)
+target_link_libraries(adv_networking_bench
+  PRIVATE
+  holoscan::core
+  holoscan::ops::advanced_network_rx
+  holoscan::ops::advanced_network_tx
+  matx::matx
+)
+
+# Configure manager support through compile definitions
+if(NOT DEFINED ANO_MGR)
+    # Initialize ANO_MGR with a default if not provided
+    set(ANO_MGR "dpdk gpunetio" CACHE STRING "Manager type(s) list")
```

Comment:
rivermax?

Reply:
not by default, since it requires licensing

Reply:
Gotcha, could be nice to have an inline comment to explain why?

Reply:
will do